src/mon/PGMonitor.cc

```diff
@@ -1473,6 +1476,16 @@ void PGMonitor::dump_pool_stats(stringstream &ss, Formatter *f, bool verbose)
           << pool_id;
       if (verbose)
 	tbl << "-";
+
+      if (pool->quota_max_objects == 0)
+        tbl << "N/A";
+      else
+        tbl << si_t(pool->quota_max_objects);
+         
+      if (pool->quota_max_bytes == 0)
+        tbl << "N/A";
+      else
+        tbl << si_t( pool->quota_max_bytes);
```

Comment:
Thanks very much for your careful check.But it look like a bug of github. My code is not have the extra whitespace.It can see in the below pull request. And i use vim to modify the code.
https://github.com/songbaisen/ceph/blob/wipdf/src/mon/PGMonitor.cc